cabal-testsuite/src/Test/Cabal/TestCode.hs

```diff
@@ -20,8 +23,10 @@ data TestCode
     = TestCodeOk
     | TestCodeSkip String
     | TestCodeKnownFail Int
-    | TestCodeUnexpectedOk
+    | TestCodeUnexpectedOk Int
```

Comment:
Could you use the `IssueID` alias here and in other places where it makes sense?

Reply:
Ah yes indeed, will do